PWGHF/HFL/TableProducer/electronSelectionWithTpcEmcal.cxx

```diff
@@ -365,18 +403,86 @@ struct HfElectronSelectionWithTpcEmcal {
           continue;
         }
 
-        isEMcal = true;
-        std::cout << " electron id  in selection" << electronId << std::endl;
-        electronSel(matchTrack.collisionId(), electronId, etaMatchTrack, phiMatchTrack, ptMatchTrack, pMatchTrack, trackRapidity, matchTrack.dcaXY(), matchTrack.dcaZ(), matchTrack.tpcNSigmaEl(), matchTrack.tofNSigmaEl(),
-                    eMatchEmcCluster, etaMatchEmcCluster, phiMatchEmcCluster, m02MatchEmcCluster, m20MatchEmcCluster, cellEmcCluster, timeEmcCluster, deltaEtaMatch, deltaPhiMatch, isEMcal);
+        /////////////////////////////////////////////////////////////            photonic electron          ///////////////////////////////////////////////////////
+
+        isLSElectronFound = 0;
+        isULSElectronFound = 0;
+        bool isLSElectron = 0;
+        bool isULSElectron = 0;
+        float invMassElectron = 0.;
+        float massLike = 0;
+        float massUnLike = 0;
+
+        for (const auto& pTrack : tracks) {
+
+          if (pTrack.globalIndex() == matchTrack.globalIndex())
+            continue;
+
+          // Apply Associated electron Selection
```

Comment:
You could comment this as "partner electron selection" instead of "Associated electron..." to match your labelling of "pTrack" and avoid any confusion